flagship.go

```diff
@@ -174,6 +178,15 @@ func (s *featureStore) ThrottleAllow(ctx context.Context, key string, hashKey io
 	return h <= t.Threshold
 
 }
+
+func (s *featureStore) ThrottleAllow(ctx context.Context, key string, hashKey io.Reader) (res bool) {
+	res = s.throttleAllow(ctx, key, hashKey)
+	if s.logger != nil {
+		s.logger.Printf("flagship.ThrottleAllow('%s') --> '%t'", key, res)
```

Comment:
I like the idea of this format string, a replacement of `==` for `-->` might make it more familiar, what do you think?